docs.openc3.com/docs/tools/cmd-tlm-server.md

```diff
@@ -35,6 +38,8 @@ Command Authority is enabled in the Admin Console and is enabled scope wide. Onc
 
 ![Command Authority](/img/cmd_tlm_server/cmd_authority.png)
 
+The other option shown in the Scope List is the Critical Command Mode. Critical commanding requires a different user to approve each command. It can either be enabled on just HAZARDOUS and RESTRICTED commands or on all manual commanding.
+
```

Comment:
Just a blurb here about Critical Command Mode because it shows up on the Scopes page. I'm not sure where else we should document this.